src/resources/dialogs/alert/alert.html

```diff
@@ -8,5 +8,7 @@
     button-text-primary.bind="buttonTextPrimary"
     button-text-secondary.bind="buttonTextSecondary"
     primary-click.call="controller.ok()"
-    secondary-click.call="controller.cancel()"></ppopup-modal>
+    secondary-click.call="controller.cancel()"
+    data.bind="data"
```

Comment:
Why are you adding `data`?  Can you please comment its purpose in the viewmodel?

Reply:
There was already a comment in the usage of the model, but I added another one in the view model